action.yml

```diff
@@ -16,11 +16,11 @@ runs:
       shell: bash
     - run: $GITHUB_ACTION_PATH/enarxbot-assigned
       shell: bash
+    - run: $GITHUB_ACTION_PATH/enarxbot-copy-labels-linked
+      shell: bash
```

Comment:
Can we print the log header here instead of polluting the scripts? If not, okay.